src/psyclone/psyir/frontend/fortran.py

```diff
@@ -83,6 +83,11 @@ def __init__(self, free_form: bool = True, ignore_comments: bool = True,
         if not self._parser:
             self._parser = ParserFactory().create(std="f2008")
         self._free_form = free_form
+        if ignore_comments and not ignore_directives:
+            raise ValueError(
+                "Setting ignore_directives to False will only have an effect"
```

Comment:
"...False in the FortranReader will only..."

Reply:
Yes, this improves the clarity, thanks. Addressed in 347af2ec2.